<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncomeDocsWindow</class>
 <widget class="QWidget" name="IncomeDocsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QToolButton" name="tbRefresh">
       <property name="toolTip">
        <string>Обновить</string>
       </property>
       <property name="toolTipDuration">
        <number>500</number>
       </property>
       <property name="text">
        <string>Обновить</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Icons/Refresh</normaloff>:/Icons/Refresh</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbNewIncomeDoc">
       <property name="toolTip">
        <string>Создать документ о приходе товара</string>
       </property>
       <property name="toolTipDuration">
        <number>500</number>
       </property>
       <property name="text">
        <string>Новый документ</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Icons/Icons/AddDocument.ico</normaloff>:/Icons/Icons/AddDocument.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbEditIncomeDoc">
       <property name="toolTip">
        <string>Изменить документ о приходе товара</string>
       </property>
       <property name="toolTipDuration">
        <number>500</number>
       </property>
       <property name="text">
        <string>Редактировать документ</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Icons/Icons/EditDocument.ico</normaloff>:/Icons/Icons/EditDocument.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbDeleteIncomeDoc">
       <property name="toolTip">
        <string>Удалить документ о приходе товара</string>
       </property>
       <property name="toolTipDuration">
        <number>500</number>
       </property>
       <property name="text">
        <string>Удалить документ</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Icons/Icons/document_delete.ico</normaloff>:/Icons/Icons/document_delete.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbApplyIncomeDoc">
       <property name="toolTip">
        <string>Применить документ о приходе товаров</string>
       </property>
       <property name="toolTipDuration">
        <number>500</number>
       </property>
       <property name="text">
        <string>Применить документ</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Icons/Icons/ok.ico</normaloff>:/Icons/Icons/ok.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <item>
      <widget class="QTableView" name="tvIncomeDocs"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
